servicetalk-client-api/src/main/java/io/servicetalk/client/api/AbstractRetryingFilterBuilder.java

```diff
@@ -286,16 +296,16 @@ public boolean isRetryable(final Meta meta, final Throwable throwable) {
                 final Executor effectiveExecutor = timerExecutor == null ?
                         requireNonNull(alternativeTimerExecutor) : timerExecutor;
                 if (exponential) {
-                    if (jitter) {
-                        return retryWithExponentialBackoffAndJitter(
-                                maxRetries, t -> true, initialDelay, effectiveExecutor);
+                    if (jitter != null) {
+                        return retryWithExponentialBackoffDeltaJitter(
```

Comment:
it may depends upon the use case whether "full" or "jitter" is required, not sure if adding additional configuration is necessary now. maybe we can defer and see if it is requested...